meson.build

```diff
@@ -4,13 +4,17 @@ project('vkd3d', ['c'], version : '1.1', meson_version : '>= 0.51', default_opti
 
 cpu_family = target_machine.cpu_family()
 
+vkd3d_compiler = meson.get_compiler('c')
+vkd3d_c_std    = 'c99'
+vkd3d_platform = target_machine.system()
+
+enable_tests             = get_option('enable_tests')
```

Comment:
weird spacing

Reply:
This lines up with enable_standalone_d3d12